packages/siop-oid4vp/lib/__tests__/IT.spec.ts

```diff
@@ -3,6 +3,8 @@ import { EventEmitter } from 'events'
 import { SigningAlgo } from '@sphereon/oid4vc-common'
 import { IPresentationDefinition } from '@sphereon/pex'
 import { CredentialMapper, IPresentation, IProofType, IVerifiableCredential, W3CVerifiablePresentation } from '@sphereon/ssi-types'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
 import nock from 'nock'
```

Comment:
why do we need to ignore it?

Reply:
because it is complaining about some typing issues, we applied this solution on all other imports like this, here it was just missing